README.md

```diff
@@ -420,9 +420,11 @@ layers:
     collectors:
       - type: className
         regex: .*Controller.*
+        predefined_delimiter: true
 ```
 
-Every classname that matches the regular expression `.*Controller.*` becomes a part of the *controller* layer.
+Every class name that matches the regular expression `.*Controller.*` becomes a part of the *controller* layer.
+Set `predefined_delimiter` to `false` to use your own delimiters and/or modifiers.
```

Comment:
This requires a practical example of how the generated regex looks like before/after.

Reply:
done